src/Policy/LimitingPolicy.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\RateLimiter\Policy;
+
+use Psr\Http\Message\ServerRequestInterface;
+
+interface LimitingPolicy
```

Comment:
```suggestion
interface LimitingPolicyInterface
```

https://github.com/yiisoft/docs/blob/master/008-interfaces.md

Reply:
((